.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "priv/cabbage"]
 	path = priv/cabbage
 	url = git@github.com:omgnetwork/specs.git
-        branch = master
\ No newline at end of file
+        branch = pgebal/add_ife_deleted_test
```

Comment:
Curious when would we move back to master?

Reply:
I'll change it before merging.